docs/getting-started-with-sylius/deployment.rst

```diff
@@ -12,123 +12,7 @@ There are many servers which you can choose for your store deployment, but in th
 Process
 -------
 
-1. Create an account on `Platform.sh <https://platform.sh/>`_
```

Comment:
/cc @CoderMaggie  should we remove this part from getting started or keep it duplicated?

Reply:
We talked that duplicate this is not good solution, link should be enough